Automattic/jetpack
# Add 'Check for Spam' button back onto the feedback page

Reverts Automattic/jetpack#27445.





#### Changes proposed in this Pull Request:

This PR brings back the 'Check for Spam' button that was removed in the PR above, and moves it below the responses table.

![Screenshot 2022-11-22 at 21 44 55](https://user-images.githubusercontent.com/8056203/203417712-44e3050a-0b82-4b00-b0b0-5fdbb9bb2cdb.png)

#### Other information:


#### Jetpack product discussion

pcsBup-oK-p2

#### Does this pull request change what data or activity we track or use?



No.

#### Testing instructions:






- Make sure you have Akismet is installed and active on your test site.
- Go to WP Admin -> Feedback and scroll down to the bottom of the page.
- There should be a 'Check for Spam' button to the right of bulk actions, below the main table.
- Clicking the 'Check for Spam' button should work the same way as it previously did, asynchronously triggering the `grunion_recheck_queue` action.
